Add the reductor class for the ROC for the clusters

cimordas opened the pull request.

wiechula commented on Modules/TPC/src/ROCReductor.cxx:
```diff
@@ -0,0 +1,86 @@
+   } // for (size_t iROC = 0; iROC < calDet.getData().size(); ++iROC)
+  } // if(pcalDet)
+} //void ROCReductor::update(o2::tpc::CalPad* pcalDet)
+*/
```
Wouldn't it make more sense to call this function from the wraper function? Otherwise I would remove the commented code.

cimordas commented:
Hello. I changed the header guard and removed completely the commented method.
I also added a small json file to run this trending and added it in the `CMakeLists.txt`.

knopers8 commented:
Is this ready to be merged? Please remove the "draft" status and the WIP tag when you want us to review & merge.

stheckel commented:
Dear Cindy, dear all,

the O2 PR https://github.com/AliceO2Group/AliceO2/pull/4006 has been merged. Thus I think, this PR is ready to be merged as well. Maybe, Cindy, you first try it locally with the most recent O2. If it works fine, you can remove the draft status and "WIP" here.

Cheers,
Stefan
